test/resources/Applicants.test.ts

```diff
@@ -19,17 +19,30 @@ function sort_by_firstname( a: Applicant, b: Applicant ) {
   return 0;
 }
 
-let modifiedApplicant = { ... exampleApplicant,
-                          firstName: "Test2" }
-
 let applicant: Applicant;
 
+async function init() {
+  applicant = await createApplicant();
+}
+
+beforeEach(() => {
+  return init();
```

Comment:
Furthermore, are those `init()` methods being called anywhere other than the `beforeEach()`? 

If they're not, we shouldn't abstract that much. I'd recommend that we eliminate the `init()` and use the code inside it directly on the `beforeEach()` for a better visibility and maintainability 💄

Reply:
Change done